### DISCUSSION TITLE
how to make it so that when intellisense imports show up, they have corresponding icon next to them?

### DISCUSSION THREAD

**Discussion by monoira** [LazyVim/LazyVim#3074] [Q&A]
## how it looks in neovim / lazy.vim 
![Screenshot_20240429_101506](https://github.com/LazyVim/LazyVim/assets/116330974/85b8f6cf-3e6c-46e3-a26e-993d3c830d00)

## how it looks in vscode
![vsc](https://github.com/LazyVim/LazyVim/assets/116330974/b5ab90ca-c1f9-4139-801d-cc47014469e3)

Plugins list:

  Loaded (48)
    ● aerial.nvim 0.79ms 󰢱 lualine.components.aerial  lualine.nvim
    ● bufferline.nvim 8.72ms  VeryLazy
    ● cmp-buffer 1.73ms  nvim-cmp
    ● cmp-nvim-lsp 1.49ms  nvim-cmp
    ● cmp-path 1.37ms  nvim-cmp
    ● cmp_luasnip 5.13ms  nvim-cmp
    ● dashboard-nvim 0.99ms  VimEnter
    ● flash.nvim 1.11ms  VeryLazy
    ● gitsigns.nvim 1.89ms 󰢱 gitsigns.config  nvim-scrollview
    ● indent-blankline.nvim 3.49ms  LazyFile
    ● lazy.nvim 46.91ms  init.lua
    ● LazyVim 7.87ms  start
    ● lualine.nvim 7.81ms  VeryLazy
    ● mason-lspconfig.nvim 0.27ms  nvim-lspconfig
    ● mason.nvim 31.31ms  nvim-lspconfig
    ● mini.ai 10.98ms  VeryLazy
    ● mini.comment 0.6ms  VeryLazy
    ● mini.indentscope 1.27ms  LazyFile
    ● mini.pairs 1.45ms  VeryLazy
    ● neoconf.nvim 0.52ms  nvim-lspconfig
    ● neodev.nvim 2.6ms  nvim-lspconfig
    ● noice.nvim 2.22ms  VeryLazy
    ● nui.nvim 0.17ms 󰢱 nui.object  noice.nvim
    ● nvim-cmp 13.1ms  InsertEnter
    ● nvim-colorizer.lua 3.09ms  start
    ● nvim-lint 1.75ms  LazyFile
    ● nvim-lspconfig 56.95ms  LazyFile
    ● nvim-navic 3.25ms 󰢱 nvim-navic  LazyVim
    ● nvim-notify 2.48ms 󰢱 notify  noice.nvim
    ● nvim-scrollview 9.17ms  start
    ● nvim-treesitter 11.96ms  VeryLazy
    ● nvim-treesitter-context 1.59ms  LazyFile
    ● nvim-treesitter-textobjects 5.38ms  nvim-treesitter
    ● nvim-ts-autotag 1.75ms  LazyFile
    ● nvim-web-devicons 0.39ms 󰢱 nvim-web-devicons  lualine.nvim
    ● onedarkpro.nvim 0.27ms colorscheme onedark  LazyVim
    ● persistence.nvim 1.08ms  BufReadPre
    ● plenary.nvim 0.34ms  vim-be-good
    ● project.nvim 8.01ms  VeryLazy
    ● smartcolumn.nvim 0.67ms  start
    ● tailwindcss-colorizer-cmp.nvim 1.17ms  nvim-cmp
    ● telescope-fzf-native.nvim 16.55ms  telescope.nvim
    ● telescope.nvim 25.27ms  Telescope
    ● tint.nvim 0.72ms  start
    ● todo-comments.nvim 2.38ms  LazyFile
    ● vim-be-good 0.59ms  start
    ● vim-illuminate 3.1ms  LazyFile
    ● which-key.nvim 0.77ms 󰢱 which-key  plugins.silicon

  Not Loaded (21)
    ○ catppuccin 
    ○ conform.nvim  <leader>cF (v)  <leader>cF  ConformInfo 
    ○ dressing.nvim 
    ○ friendly-snippets  LuaSnip 
    ○ headlines.nvim  markdown  rmd  org  norg 
    ○ LuaSnip  <tab> (s)  <s-tab> (s)  <s-tab> (i)  <tab> (i) 
    ○ markdown-preview.nvim  <leader>cp  MarkdownPreview  MarkdownPreviewStop  MarkdownPreviewToggle 
    ○ mini.bufremove  <leader>bd  <leader>bD 
    ○ mini.surround  gsF  gsr  gsn  gsf  gsa (v)  gsh  gsd  gsa 
    ○ neo-tree.nvim  <leader>ge  <leader>be  <leader>fe  <leader>fE  <leader>e  <leader>E  Neotree 
    ○ nvim-silicon  Silicon 
    ○ nvim-spectre  <leader>sp  <leader>sP  <leader>sr  <leader>sR  Spectre 
    ○ nvim-ts-context-commentstring 
    ○ obsidian.nvim  markdown 
    ○ omnisharp-extended-lsp.nvim 
    ○ SchemaStore.nvim 
    ○ toggleterm.nvim  <C-\> 
    ○ tokyonight.nvim 
    ○ trouble.nvim  <leader>xQ  ]q  [q  <leader>xx  <leader>xX  <leader>xL  TroubleToggle  Trouble 
    ○ vim-startuptime  StartupTime 
    ○ vim-table-mode  <leader>sx

**Comment by dpetka2001** [comment]
You should take a look at `nvim-cmp` [Wiki](https://github.com/hrsh7th/nvim-cmp/wiki/Menu-Appearance#how-to-get-types-on-the-left-and-offset-the-menu). It has quite a few recipes from users that you can take a look at and be inspired from.

**Reply by monoira** [reply]
I'm very new. I need easier answer. I will take a look at that  later when I'm not noob anymore

**Reply by dpetka2001** [reply]
I don't have an easier answer, because I have never bothered changing the default appearance. Sorry I can't help further.
